src/vivarium_gates_mncnh/data/loader.py

```diff
@@ -577,6 +548,35 @@ def load_no_acs_relative_risk(
     return utilities.set_non_neonnatal_values(no_acs_rr, 1.0)
 
 
+def load_no_acs_paf(
+    key: str, location: str, years: Optional[Union[int, str, List[int]]] = None
+) -> float:
+    p_CPAP_BEmONC = get_data(data_keys.NO_CPAP_RISK.P_CPAP_BEMONC, location, years)
+    p_CPAP_CEmONC = get_data(data_keys.NO_CPAP_RISK.P_CPAP_CEMONC, location, years)
+    rr_no_CPAP = get_data(data_keys.NO_CPAP_RISK.RELATIVE_RISK, location, years)
+    rr_no_ACS = get_data(data_keys.NO_ACS_RISK.RELATIVE_RISK, location, years)
+    p_BEmONC_given_facility = data_values.DELIVERY_FACILITY_TYPE_PROBABILITIES[location][
```

Comment:
Not important to the code, but I wonder if we should rename "p_bemonc" to "bemonc_facility_fraction" or something like that so that it's more clear and we don't accidentally use it as the BEMONC rate in the overall population

Reply:
Also, do we need to update this how we've defined these parameters in the PAF equations for all of the other interventions that depend on facility type given that we've changed how we have defined these inputs?